services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/command/flow/VerificationDispatchCommand.java

```diff
@@ -95,8 +96,8 @@ private void launchSubCommands(List<Optional<Command>> subCommands) {
         return Optional.of(new VerificationListenCommand(getContext(), verificationRequest));
     }
 
-    private boolean isOwnSwitch(String switchId) {
-        DatapathId dpId = DatapathId.of(switchId);
+    private boolean isOwnSwitch(SwitchId switchId) {
+        DatapathId dpId = DatapathId.of(switchId.toString());
```

Comment:
there is `DatapathId.of(Long)` method exist, no need for String conversion loop.